test/acceptance_with_python/test_collection.py

```diff
@@ -0,0 +1,37 @@
+import weaviate
+import weaviate.classes as wvc
+
+def test_collection_casing() -> None:
+    with weaviate.connect_to_local() as client:
+        # Goal: Collection create/get/delete should automatically transform class name to GQL (first letter caps).
+
+        # create collection with all "lower" case -> GQL ("Testcollectioncase")
```

Comment:
I think our python client already "transform" the class name before calling GET/DELETE endpoint.

But still added to lock the behaviour.

Unit test cases(below) locks this behaviour at API level (if you remove the changes, those test fails)